tests/robot-tests/tests/admin/bau/release_content_authoring.robot

```diff
@@ -241,8 +241,10 @@ Switch back to analyst1 to resolve second text block
     ${comment}=    user gets unresolved comment    Test comment 3    ${block}
     ${author}=    get child element    ${comment}    testid:comment-author
     user waits until element contains    ${author}    Bau1 User1
+
     # resolve the comment left by bau1
-    user clicks button    Resolve    ${comment}
+    user sets focus to element    xpath://button[text()="Resolve"]    ${block}
```

Comment:
Perhaps we could put this focus step in `user clicks element`?

```robot
user scrolls to element    ${element}
wait until element is enabled    ${element}
user sets focus to element   ${element}
click element    ${element}
```

Reply:
This is is pretty weird, why would the original version work and not this version? Bizarre!

Reply:
Not too sure. I can remember having issues getting this test suite to pass when it was initially created due to clicks being intercepted when resolving comments